src/converters/freetradeConverter.test.ts

```diff
@@ -6,6 +6,14 @@ describe("freetradeConverter", () => {
 
   beforeEach(() => {
     jest.spyOn(console, "log").mockImplementation(jest.fn());
+    // Mock YahooFinanceService - we don't want to make real requests and don't care about the result.
```

Comment:
I certainly do see the appeal of this, as it does shorten the test time significantly.  I also agree with your statement that the YahooFinance converter should get it's own test file. I haven't gotten to that just yet 😄

However I feel this approach might be a little simplistic regarding testing the converters. I think the coverage of this approach will drop below the current %. The "beauty" (or not, depending on how you look at it 😉) of the current approach is that it magically covers (almost) all branches in the converter. I feel a better (but for sure more time consuming approach) would be to register all the current calls that are done to Yahoo Finance, register all the responses and put those in a (generic) mock file that can be used in all of the converter tests. This way you have the benefit of mocking (and not calling Yahoo Finance, thus making it faster) and also the guaranteed repeatability of the test.

Reply:
Added #58 for this purpose. When I have some time I will make a start with this.